server/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const path = require('path');
+const paths = require('../config/paths');
```

Comment:
🧐
Я видел, что ты повырубал правила еслинта насчёт импортом, но их было бы хорошо затащить, чтобы заборчик от нормальных импортов отделять
```suggestion
const path = require('path');

const paths = require('../config/paths');
```

Reply:
ага надо будет разобраться